guides/01_getting-started/03_sharing-your-app.md

```diff
@@ -145,7 +145,7 @@ This will add and document the endpoint `/api/addition/` to the automatically ge
 
 *Note*: For Gradio apps in which [queueing is enabled](https://gradio.app/guides/key-features#queuing), users can bypass the queue if they make a POST request to your API endpoint. To disable this behavior, set `api_open=False` in the `queue()` method. To disable the API page altogether, set `show_api=False` in `.launch()`.
 
-## Authentication
+## Password-protected app
```

Comment:
Doing it this way just so the old anchor links (which we've shared in other places) continue to work

```suggestion
## Authentication

### Password-protected app
```